code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/AutoCarouselNotificationBuilder.kt

```diff
@@ -61,24 +72,6 @@ internal object AutoCarouselNotificationBuilder {
             packageName
         )
 
-        // fallback to a basic push template notification builder if less than 3 images were able to be downloaded
-        if (downloadedImageCount < PushTemplateConstants.DefaultValues.CAROUSEL_MINIMUM_IMAGE_COUNT) {
```

Comment:
Why did we remove this check for Auto carousel? If downloaded images are less than 3, we still need to fallback to basic template right?

Reply:
nit: Let's also verify/inform PM about this behavior change